ocaml/idl/datamodel.ml

```diff
@@ -1421,7 +1422,12 @@ let _ =
 
   error Api_errors.pvs_server_address_in_use ["address"]
     ~doc:"The address specified is already in use by an existing PVS_server object"
-    ()
+    ();
+
+  (*Feature errors*)
+  error Api_errors.multiple_feature_instance ["feature"; "host"]
```

Comment:
Theoretically speaking it could be more than two - is is more expensive to delete all and create a new one?